qa/L0_sequence_batcher/sequence_batcher_test.py

```diff
@@ -1790,9 +1791,8 @@ def test_backlog_same_correlation_id_no_end(self):
                             1002,
                             (None, None),
                             # (flag_str, value, pre_delay_ms)
-                            (("start", 11, None), (None, 12, None),
-                             (None, 12, None), ("end", 13, None)),
```

Comment:
Why switching the order. You are changing the test behavior so something seems wrong

Reply:
Please take a look at the above comment.